data-migrations/2024-09-05-0945-seed-tags.sql

```diff
@@ -0,0 +1,32 @@
+INSERT INTO tags (name, description, createdById, modifiedById)
```

Comment:
@briri - where do these tags come from? They are different than the Themes, and I couldn't remember if we had another document listing these out somewhere.

Reply:
Thanks for that info @briri.